app/models/hour.rb

```diff
@@ -0,0 +1,57 @@
+# == Schema Information
+#
+# Table name: hours
+#
+#  id          :integer          not null, primary key
+#  project_id  :integer          not null
+#  category_id :integer          not null
+#  user_id     :integer          not null
+#  hours       :integer          not null
+#  date        :date             not null
+#  created_at  :datetime
+#  updated_at  :datetime
+#  description :string(255)
+#  billed      :boolean
+#
+
+class Hour < Entry
+	include Twitter::Extractor
+
+	audited allow_mass_assignment: true
+
+	belongs_to :project, touch: true
+	belongs_to :category
+	has_one :client, through: :project
+	belongs_to :user, touch: true
+	has_many :taggings, inverse_of: :value
+	has_many :tags, through: :taggings
+
+	validates :user, :project, :category, :date, presence: true
+	validates :value, presence: true,
+	                numericality: { greater_than: 0, only_integer: true }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Tab detected.